config/locales/client.en.yml

```diff
@@ -1020,3 +1020,8 @@ en:
       body_html: "<p>Yikes, did we do something wrong? We're sorry if you had a bad time in the new Loomio.</p>"
       give_feedback: "Before you go please consider sharing your feedback so we can improve."
       button_text: 'Switch back to the old Loomio'
+
+  global:
+    messages:
+      app_update: 'Loomio has been updated!'
+      get_the_latest: 'Get the latest'
```

Comment:
Or just "Reload".

I think "Loomio as been updated" is particularly confusing.. because that is something I would expect to see after I have reloaded.